cmd/enduro/main.go

```diff
@@ -335,6 +337,14 @@ func main() {
 		}
 	}
 
+	// Set up reports bucket.
+	reportsBucket, err := bucket.NewWithConfig(ctx, cfg.Storage.Reports.Bucket)
+	if err != nil {
+		logger.Error(err, "Error setting up reports bucket.")
+		os.Exit(1)
+	}
+	defer reportsBucket.Close()
+
```

Comment:
We discussed the proliferation of buckets on the ingest side and decided to use a single internal bucket for most cases. We already have an internal bucket on the storage side that is only use for AIP review, I think we should use that one.

In some cases MinIO access may be restricted, so we will need to provide a link to the report in the dashboard. In another PR, as we are discussing in the standup.

Reply:
Not the ingest internal bucket, the storage internal bucket/location/storage. It's set up right above:

https://github.com/artefactual-sdps/enduro/blob/main/cmd/enduro/main.go#L245-L251